src/api/svg.ts

```diff
@@ -0,0 +1,537 @@
+        x: options.x + element.svgAttributes.x1,
+        y: options.y - element.svgAttributes.y1,
+      },
+      end: {
+        x: options.x + element.svgAttributes.x2,
+        y: options.y - element.svgAttributes.y2,
+      },
+      thickness: element.svgAttributes.strokeWidth,
+      color: element.svgAttributes.stroke,
+      opacity: element.svgAttributes.strokeOpacity,
+      lineCap: element.svgAttributes.strokeLineCap,
+    });
+  },
+  async path(element) {
+    page.drawSvgPath(element.svgAttributes.d, {
+      x: options.x + element.svgAttributes.x,
+      y: options.y - element.svgAttributes.y,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      scale: element.svgAttributes.scale,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async image(element) {
+    page.drawImage(await page.doc.embedPng(element.svgAttributes.src), {
+      x: options.x + element.svgAttributes.x,
+      y: options.y - element.svgAttributes.y,
+      width: element.svgAttributes.width,
+      height: element.svgAttributes.height,
+      opacity: element.svgAttributes.fillOpacity,
+      xSkew: element.svgAttributes.skewX,
+      ySkew: element.svgAttributes.skewY,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async rect(element) {
+    if (!element.svgAttributes.fill && !element.svgAttributes.stroke) return;
+    page.drawRectangle({
+      x: options.x + element.svgAttributes.x,
+      y: options.y - element.svgAttributes.y,
+      width: element.svgAttributes.width,
+      height: element.svgAttributes.height,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      xSkew: element.svgAttributes.skewX,
+      ySkew: element.svgAttributes.skewY,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async ellipse(element) {
+    page.drawEllipse({
+      x: options.x + element.svgAttributes.x + element.svgAttributes.cx,
+      y: options.y - element.svgAttributes.y - element.svgAttributes.cy,
+      xScale: element.svgAttributes.rx,
+      yScale: element.svgAttributes.ry,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async circle(element) {
+    page.drawCircle({
```

Comment:
Circle should be rendered as ellipsis as you cannot be sure that the scale on x equals the scale on y.